website/components/GoalForm/GoalForm.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="container ">
+    <form action="">
+      <h2 class="text-center">
+        Create a New Goal
+      </h2>
+      <InputField
+        id="task-name"
+        label="Goal Title"
+        type="text"
+        name="task-name"
+        required
+        placeholder="eg: Testing setup for dashboard"
+        autofocus
+      />
+      <InputField
+        id="title"
+        label="Task Name"
+        type="text"
+        name="title"
+        required
+        placeholder="eg: Testing setup for dashboard"
+      />
+      <InputField
+        id="links"
+        label="Links"
+        type="text"
+        name="links"
+        required
+        placeholder="Comma separated links"
+      />
+      <div class="input-container">
+        <SelectOption
+          label="User Role"
+          name="user-role"
+          required
+          :options="usernameCategoryOptions"
+        />
+        <InputField
+          id="username"
+          label="Username"
+          type="text"
+          name="username"
+          required
+          placeholder="eg: sahsisunny"
+        />
```

Comment:
We won't need this as the data is provided by PopupAssignee

Reply:
removed